funcs_utils.php

```diff
@@ -407,3 +428,38 @@ function current_branch_cms_major(
     }
     return (string) $cmsMajor;
 }
+
+function setup_directories($input) {
```

Comment:
I moved code here for functions setup_directories() + filtered_modules() from update_command.php to prevent code duplication because they're also used in the new labels_command.php file